### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-* [TestPage.aspx](./CS/TestPage.aspx) (VB: [TestPage.aspx](./VB/TestPage.aspx))
-* [TestPage.aspx.cs](./CS/TestPage.aspx.cs) (VB: [TestPage.aspx.vb](./VB/TestPage.aspx.vb))
-<!-- default file list end -->
-# How to show the ASPxLoadingPanel while the content is loading inside the ASPxPopupControl
+# Popup Control for ASP.NET Web Forms - How to show the loading panel on the content's loading
```
Comment (arminal) [inline Readme.md]:
```suggestion
# Popup Control for ASP.NET Web Forms - How to show the loading panel when loading content
```

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e1945/)**
 <!-- run online end -->
 
+This example demonstrates how use the popup control's client-side functionality to display the loading panel in a pop-up window when the content loading is slow.
```
Comment (arminal) [inline Readme.md]:
```suggestion
This example demonstrates how use the popup control's client-side functionality to display the loading panel in a pop-up window when content is loading slowly.
```

File: Readme.md [hunk 3]
```diff
@@ -1,24 +1,41 @@
...
+
+![Loading panel in pop-up window](loadingPanelInPopup.png)
 
-<p>If a content of the ASPxPopupControl is loaded slowly, you can show a loading panel reflecting this process.</p>
+## Overview
 
-<br/>
+The main idea is to call the popup control's client-side [GetContentIFrame](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.GetContentIFrame) method to get the control's IFrame. Then use the loading panel's [ShowInElement](https://docs.devexpress.com/AspNet/js-ASPxClientLoadingPanel.ShowInElement(htmlElement)) method to display the panel within the IFrame. When the content is loaded, hide the loading panel.
```
Comment (arminal) [inline Readme.md]:
```suggestion
The main idea [idea? did you mean action?] is to call the popup control's client-side [GetContentIFrame](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.GetContentIFrame) method to get the control's IFrame. Then use the loading panel's [ShowInElement](https://docs.devexpress.com/AspNet/js-ASPxClientLoadingPanel.ShowInElement(htmlElement)) method to display the panel within the IFrame. When content is loaded, hide the loading panel.
```

**Comment by arminal** [comment]
#Articles_Grammar
No articles with uncountables

File: Readme.md (hunk 3 above)
Comment (elenapeskova) [reply Readme.md]:
```suggestion
The main idea is to call the popup control's client-side [GetContentIFrame](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.GetContentIFrame) method to get the control's IFrame. Then use the loading panel's [ShowInElement](https://docs.devexpress.com/AspNet/js-ASPxClientLoadingPanel.ShowInElement(htmlElement)) method to display the panel within the IFrame. When content is loaded, hide the loading panel.
```

**Pull request merged**